.gitignore

```diff
@@ -0,0 +1,2 @@
+# Dependency directories
+node_modules/
```

Comment:
Could you add a new line at the end of the file here please? You can turn it on in VSCode as well to automatically add it:

![image](https://user-images.githubusercontent.com/4377076/68587386-e99d1e00-047e-11ea-94cf-d77dac77538d.png)

Reply:
done - thank you.